code/modules/surgery/eye_surgery.dm

```diff
@@ -1,6 +1,5 @@
 /datum/surgery/eye_surgery
 	name = "Eye surgery"
-	requires_bodypart_type = NONE
```

Comment:
see above but `monkestation/code/modules/surgery/eye_surgery.dm` instead

Reply:
Alright so theres one issue with this change for eye surgery specifically

Currently, if you have it defined as NONE or BODYTYPE_ORGANIC it will have both eye surgeries show up on robotic head types.

I don't know what causes it currently. But that's the current issue for it.
Ear surgery which uses the same functional bulk of code, but targets the heads instead of the eyes, does not suffer from this issue.

Reply:
hmmm weird. likely not related to this PR tho?